### ISSUE TITLE
TypeError: Cannot read property 'language' of null

### ISSUE THREAD

**Issue by Meldiron** [PrismarineJS/mineflayer#562]
Error:

```
MacBook-Pro-uzivatela-macbook:minecraftBot Meldiron$ node app.js 
/Users/Meldiron/Desktop/minecraftBot/node_modules/mineflayer/lib/chat_message.js:8
  defaultLang = mcData.language
                      ^

TypeError: Cannot read property 'language' of null
    at new loader (/Users/Meldiron/Desktop/minecraftBot/node_modules/mineflayer/lib/chat_message.js:8:23)
    at Client.bot._client.on (/Users/Meldiron/Desktop/minecraftBot/node_modules/mineflayer/lib/plugins/blocks.js:274:21)
    at emitTwo (events.js:106:13)
    at Client.emit (events.js:191:7)
    at Parser.deserializer.on (/Users/Meldiron/Desktop/minecraftBot/node_modules/minecraft-protocol/src/client.js:102:12)
    at emitOne (events.js:96:13)
    at Parser.emit (events.js:188:7)
    at addChunk (/Users/Meldiron/Desktop/minecraftBot/node_modules/readable-stream/lib/_stream_readable.js:284:12)
    at readableAddChunk (/Users/Meldiron/Desktop/minecraftBot/node_modules/readable-stream/lib/_stream_readable.js:271:11)
    at Parser.Readable.push (/Users/Meldiron/Desktop/minecraftBot/node_modules/readable-stream/lib/_stream_readable.js:238:10)
    at Parser.Transform.push (/Users/Meldiron/Desktop/minecraftBot/node_modules/readable-stream/lib/_stream_transform.js:146:32)
MacBook-Pro-uzivatela-macbook:minecraftBot Meldiron$ 
```

My code:
```
var mineflayer = require('mineflayer');
mineflayer.createBot({
    host: "89.203.249.22", // optional
    port: 25565,       // optional
    username: randName()
});

function randName() {
    return "meldiron" + (Math.floor(Math.random() * 10) + 1);
}
```

**Comment by Meldiron** [comment]
Same error with your example code echo.js ( https://github.com/PrismarineJS/mineflayer/blob/master/examples/echo.js )

command i ran: node app.js 89.203.249.22 25565 meltest

**Comment by rom1504** [comment]
What version of Minecraft is the server ?

**Comment by rom1504** [comment]
Nevermind, there is the bug https://github.com/PrismarineJS/mineflayer/blob/master/lib/plugins/blocks.js#L6
The version needs to be passed to chat_message
Can you look at it @plexigras ?

**Comment by rom1504** [comment]
fixed by #564
